BESMod/Systems/Hydraulical/Generation/GasBoiler.mo

```diff
@@ -146,4 +148,9 @@ equation
       index=1,
       extent={{-6,3},{-6,3}},
       horizontalAlignment=TextAlignment.Right));
+  annotation (Documentation(info="<html>
+<h4>Bottom-up parameters</h4>
+<p><br>The value for dTTra_nominal is based on EN 303-1:2017, which requires a temperature spread between 10 and 20 K. </p>
```

Comment:
You can check e.g. Verordnung (EU) Nr. 813/2013 additionally

Reply:
Thanks, but this does not add information :)